core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -1355,11 +1389,10 @@ private String buildInOperator(int size) {
         if (size == 1) {
             return "(?)";
         }
-        StringBuilder builder = new StringBuilder("(");
-        for (int i = 0; i < size - 1; i++) {
-            builder.append("?,");
-        }
-        builder.append("?)");
+        StringBuilder builder =
+                new StringBuilder("(")
+                        .append(StringUtils.join(Collections.nCopies(size, "?"), ","))
```

Comment:
I mostly looked into removing conditional code. Also, level 15 will let us use try-with-resource that would have been very useful for that other PR.

Anyway, out of scope for this PR. Does it look good or any other change needed?